optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-client/src/main/java/org/optaplanner/workbench/screens/solver/client/editor/TerminationTreeItemContentViewImpl.html

```diff
@@ -27,48 +32,65 @@
                 <span id="dropdownMenuList" />
             </div>
 
-            <div id="terminationCompositionStyleDiv" class="col-xs-3 col-sm-3 col-md-3" >
+            <div id="terminationCompositionStyleDiv" class="" style="display: inline-block;">
                 <select id="terminationCompositionStyleSelect" class="form-control" style="width: auto">
                     <option id="orOption" value="or" data-i18n-key="Or"></option>
                     <option id="andOption" value="and" data-i18n-key="And"></option>
                 </select>
             </div>
 
-            <div id="dropDownHelpIcon" class="col-xs-1 col-sm-1 col-md-1 uf-help-icon" >
+            <div id="dropDownHelpIcon" class="uf-help-icon">
             </div>
-
-            <label id="formLabel">
-            </label>
         </div>
 
         <div id="terminationValuesDiv" class="col-xs-7 col-sm-7 col-md-7">
-            <div id="timeSpentDiv">
+            <div id="timeSpentDiv" style="white-space: normal;">
                 <form>
-                    <input id="daysSpentInput" type="number" min="0" max="9223372036854775807" style="width: 50px;" />
-                    <label for="daysSpentInput" data-i18n-key="Days"></label>
-                    <input id="hoursSpentInput" type="number" min="0" max="9223372036854775807" style="width: 50px;"/>
-                    <label for="hoursSpentInput" data-i18n-key="Hours"></label>
-                    <input id="minutesSpentInput" type="number" min="0" max="9223372036854775807" style="width: 50px;"/>
-                    <label for="minutesSpentInput" data-i18n-key="Minutes"></label>
-                    <input id="secondsSpentInput" type="number" min="0" max="9223372036854775807" style="width: 50px;"/>
-                    <label for="secondsSpentInput" data-i18n-key="Seconds"></label>
-                    <input id="millisecondsSpentInput" type="number" min="0" max="9223372036854775807" style="width: 50px;"/>
-                    <label for="millisecondsSpentInput" data-i18n-key="Milliseconds"></label>
+                    <div style="display: inline-block">
+                        <input id="daysSpentInput" type="number" min="0" max="9223372036854775807" style="width: 50px;" />
```

Comment:
@ge0ffrey I don't think you can do that easily just using plain css. The solution might be to set the min/max values from the underlying view bean after injection is complete.